web/src/main.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-require('./css/style.css');
+require('./css/main.scss');
+require('./css/_popup.scss');
```

Comment:
if you `@import` the popup and dashboard css files in main.scss, you won't need to require them here again